configs/manager/sample.toml

```diff
@@ -243,3 +243,7 @@ enhanced-aiomonitor-task-info = false
 
 # Periodically sync container stat from Redis to the kernels.last_stat column.
 periodic-sync-stats = false
+
+
+[license]
+addr = { host = "127.0.0.1", port = 6099 }
```

Comment:
I haven't found the config code for the license, could you point me to where it is declared?